core/src/main/java/com/datastax/oss/driver/internal/core/metadata/DefaultNode.java

```diff
@@ -97,19 +98,19 @@ public EndPoint getEndPoint() {
     return Optional.ofNullable(listenAddress);
   }
 
-  @NonNull
+  @Nullable
```

Comment:
This was my own mistake, I guess, when I added annotations to these classes. The interface methods were marked nullable, but the implementations here were mistakenly marked non-null.